### PR TITLE
Update managing-access-to-self-hosted-runners-using-groups.md

### LINKED ISSUE #9067: Add documentation for runnergroup command line option
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's Code of Conduct.

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups

### What part(s) of the article would you like to see updated?

Add a paragraph explaining how to use the configuration script (`config.{sh|cmd}`) to create a runner in a specific runner group.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by fhammerl**
Describes how you can create a self-hosted runner in a runnergroup

### Why:

Closes https://github.com/github/docs/issues/9067
Closes https://github.com/github/c2c-actions-runtime/issues/872

### What's being changed:

I'm adding a description of how to create a runner in a specific runner group

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md [hunk 1]
```diff
@@ -100,10 +100,19 @@ When creating a group, you must choose a policy that defines which organizations
 You can update the access policy of a runner group, or rename a runner group.
 
 {% data reusables.github-actions.self-hosted-runner-configure-runner-group-access %}
+    
+## Creating a self-hosted runner in a group
+You can use the configuration script on the self-hosted runner to create it in a runner group. For example, this command creates a self-hosted runner in the runner group `rg-runnergroup`.
+
+```sh
+./config.sh --url $org --token $token --runnergroup rg-runnergroup
```
Comment (thboop) [inline content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md:108]: --url $org? Shouldn't that be the runner registration url

**Comment by MelnikovAG** [comment]
Yo

File: content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md (hunk 1 above)
Comment (fhammerl) [reply content/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups.md:108]: You're right, the way I've written it is not precise enough. It needs to be the url of the enterprise or org in which you're registering the runner.

How about being very specific:

```sh
./config.sh --url $org_or_enterprise_url --token $token --runnergroup rg-runnergroup
```

**Comment by ramyaparimi** [comment]
@fhammerl Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by lucascosti** [comment]
Drive by comment: we should maybe consider editing the content in the [Adding self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners) article, rather than creating a new section on adding runners here?

**Review by Saulsbert38 — DISMISSED** [review]

**Review by martin389 — APPROVED** [review]

**Pull request merged**